ui/app/components/ui/fiat-value.js

```diff
@@ -38,31 +37,33 @@ function fiatDisplay (fiatDisplayNumber, fiatSuffix, styleOveride = {}) {
   const { fontSize, color, fontFamily, lineHeight } = styleOveride
 
   if (fiatDisplayNumber !== 'N/A') {
-    return h('.flex-row', {
-      style: {
-        alignItems: 'flex-end',
-        lineHeight: lineHeight || '13px',
-        fontFamily: fontFamily || 'Montserrat Light',
-        textRendering: 'geometricPrecision',
-      },
-    }, [
-      h('div', {
-        style: {
-          width: '100%',
-          textAlign: 'right',
-          fontSize: fontSize || '12px',
-          color: color || '#333333',
-        },
-      }, fiatDisplayNumber),
-      h('div', {
-        style: {
-          color: color || '#AEAEAE',
-          marginLeft: '5px',
-          fontSize: fontSize || '12px',
-        },
-      }, fiatSuffix),
-    ])
+    return (
+      <div
+        className="flex-row"
+        style={{
+          alignItems: 'flex-end',
+          lineHeight: lineHeight || '13px',
+          fontFamily: fontFamily || 'Montserrat Light',
+          textRendering: 'geometricPrecision',
+        }}>
+        <div
+          style={{
+            width: '100%',
+            textAlign: 'right',
+            fontSize: fontSize || '12px',
+            color: color || '#333333',
+          }}>{fiatDisplayNumber}</div><div
```

Comment:
:eyes:  That is some interesting formatting

Reply:
Whoops, my bad 😅